e2e/fixtures/fdb_cluster.go

```diff
@@ -119,6 +119,23 @@ func (fdbCluster *FdbCluster) Update() error {
 	return fdbCluster.getClient().Update(ctx.Background(), fdbCluster.cluster)
 }
 
+// UpdateNode update node definition
+func (fdbCluster *FdbCluster) UpdateNode(node *corev1.Node) error {
+	return fdbCluster.getClient().Update(ctx.Background(), node)
+}
+
+// GetNode return Node with the given name
+func (fdbCluster *FdbCluster) GetNode(name string) *corev1.Node {
+	// Retry if for some reasons an error is returned
+	node := &corev1.Node{}
+	gomega.Eventually(func() error {
+		return fdbCluster.getClient().
+			Get(ctx.TODO(), client.ObjectKey{Name: name, Namespace: fdbCluster.Namespace()}, node)
```

Comment:
```suggestion
			Get(ctx.TODO(), client.ObjectKey{Name: name}, node)
```

A node has no namespace.